models/job.go

```diff
@@ -329,7 +329,9 @@ type HTTPDependency struct {
 // JobService provides a high-level operations on DAGs
 type JobService interface {
 	// Create constructs a Job and commits it to a storage
-	Create(context.Context, NamespaceSpec, JobSpec) error
+	Create(context.Context, NamespaceSpec, JobSpec) (*JobSpec, error)
```

Comment:
Why do we need to return it as pointer? When I check it in the implementation, we will get the value again. If possible, please make it consistent.